src/styles/BookCard.scss

```diff
@@ -52,8 +53,63 @@
 
 .book__left-h3,
 .book__left-p,
-.book__left-li,
+.book__left-button,
 .book__left-p-2,
 .book__right-h3 {
   color: map-get($colors, faded-text);
 }
+
+.book__left-button {
+  border: none;
+  background-color: map-get($colors, primary-white);
+  color: map-get($colors, secondary-blue);
+  font-family: $primary-font;
+  font-weight: 100;
+}
+
+.book__left-button:hover {
+  cursor: pointer;
+  color: map-get($colors, primary-black);
+}
+
+.book__left-vertical {
+  width: 0.125rem;
+  height: 1.5rem;
+  background-color: #e8e8e8;
+  margin: 0 1rem;
+}
+
+.book__left-percentage > * {
+  padding: 0 1rem;
+  font-size: 1rem;
+  font-weight: 100;
+}
+
+.book__left-circle {
+  border-radius: 50%;
+  width: 4.25rem;
+  height: 4.25rem;
+  border: 5px solid #307bbe;
+  border-left-color: #e8e8e8;
+  transform: rotate(45deg);
+}
+
+.book__left-description {
+  font-size: 0.8rem;
+}
+
+.book__left-description > * {
+  margin: 0.5rem 0;
+}
+
+.book__right-h3-2 {
+  font-weight: 100;
+}
+
+.book__right > * {
+  margin: 0.5rem 0;
+}
+
+.book__left-h2-2 {
+  font-size: 1.5rem;
```

Comment:
- [ ] Great work implementing your design in line with the project requirements. However, there're few things you need to work on before your project could be approved. Kindly ensure that you produce a close replica of the Zeplin design. This includes background color, text-color, margin, padding among others. Look at the both screenshots below, check the areas I marked with red lines and walk your desgin alongside the Zeplin design. Trust me, you're almost there.

### Your Design
![your-design](https://github.com/J-C-S-V/bookstore/assets/26597432/af310a58-7c87-40b9-b125-c8c568225cee)

### Zeplin Design 
![temp-design](https://github.com/J-C-S-V/bookstore/assets/26597432/7e68a3e6-58f9-44bb-b3f5-8f8bb10df078)